### PR TITLE
Minor optimizations and cleaning up.

### PR DESCRIPTION
**Pull request by gsquaredxc**
Cleaning up some code to help slightly improve performance, is draft for now. Mainly was to help readability but some things can help make the code faster.

### CODE REVIEW

File: patches/net/minecraft/server/MinecraftServer.java.patch [hunk 1]
```diff
@@ -9,7 +9,15 @@
  import com.google.common.collect.Lists;
  import com.google.common.collect.Queues;
  import com.google.common.util.concurrent.Futures;
-@@ -86,6 +89,14 @@
+@@ -33,7 +36,6 @@
+ import java.util.concurrent.Callable;
+ import java.util.concurrent.Executors;
+ import java.util.concurrent.FutureTask;
+-import java.util.function.Supplier;
```
Comment (Earthcomputer) [inline patches/net/minecraft/server/MinecraftServer.java.patch:16]: Do not add patches to remove warnings from vanilla code. None of these changes to this class are actual optimizations.

**Review by Earthcomputer — CHANGES_REQUESTED** [review]
The rest of the PR is IMO okay as just a general cleanup PR, none of this is optimizations though.

File: patches/net/minecraft/server/MinecraftServer.java.patch (hunk 1 above)
Comment (gsquaredxc) [reply patches/net/minecraft/server/MinecraftServer.java.patch:16]: Just to double check, this is just the import right? I also removed some unused variables that were just there for no reason.

Comment (Earthcomputer) [reply patches/net/minecraft/server/MinecraftServer.java.patch:16]: It is almost everything you changed in this class. None of them are optimizations. The variables are generated by the decompiler and will be removed by the JIT if not the compiler itself. The vast majority of this PR is mostly just style fixes and not effective optimizations like you are aiming. I am fine with style fixes (although it's ultimately up to Xcom), however don't fix the style of the decompiler.

**Review by Earthcomputer — CHANGES_REQUESTED** [review]
Changes to vanilla code need carpet rules. Please limit the scope of this PR to cleaning up carpet code only.

**Review by X-com — CHANGES_REQUESTED** [review]
Agree with earth here. That's just some mess fern flower spits out. I doubt it influences runtime operation by more then few nanoseconds, but on the other hand it generates another patch and possibly causes future problems when binary's cause issues again. Just leave vanilla fields totally alone.

**Pull request closed without merging**